pydrive2/test/test_apiattr.py

```diff
@@ -1,34 +1,39 @@
 import unittest
+import pytest
 
 from pydrive2.auth import GoogleAuth
 from pydrive2.drive import GoogleDrive
+from pydrive2.test.test_util import pydrive_retry, setup_credentials
 
+
+@pytest.mark.automated
 class ApiAttributeTest(unittest.TestCase):
     """Test ApiAttr functions.
     """
-    ga = GoogleAuth('settings/test1.yaml')
-    ga.LocalWebserverAuth()
-    first_file = 'a.png'
-    second_file = 'b.png'
+
+    @classmethod
+    def setup_class(cls):
+        setup_credentials()
 
     def test_UpdateMetadataNotInfinitelyNesting(self):
         # Verify 'metadata' field present.
         self.assertTrue(self.file1.metadata is not None)
-        self.file1.UpdateMetadata()
-        self.file1.UpdateMetadata()
+        pydrive_retry(lambda: self.file1.UpdateMetadata())
```

Comment:
can we just pass `self.file1.UpdateMetadata` instead of a lambda?

Reply:
thanks, works good without lambda as well; Will simplify for all places without params in function call.